mifospay/src/main/java/org/mifos/mobilewallet/mifospay/standinginstruction/ui/SIDetailsActivity.kt

```diff
@@ -41,6 +47,7 @@ class SIDetailsActivity : BaseActivity(), StandingInstructionContract.SIDetailsV
     private lateinit var mStandingInstructionPresenter:
             StandingInstructionContract.StandingInstructorDetailsPresenter
 
+    @SuppressLint("NewApi")
```

Comment:
Any justification for this suppressing lint for this?

Reply:
Yeah i have removed that right now,That was in accordance to the previous commit